aws/cloudformation/iam.yml.erb

```diff
@@ -563,16 +591,18 @@ roles.each do |name, config| %>
               - arn:aws:s3:::cdo-animation-library/*
               - arn:aws:s3:::cdo-sound-library/*
 
-  DenySecretsPolicy:
+  EngineerSecretsPolicy:
     Type: AWS::IAM::ManagedPolicy
     Properties:
-      ManagedPolicyName: DenySecretsPolicy
+      ManagedPolicyName: EngineerSecretsPolicy
       Path: /engineering/
       PolicyDocument:
         Version: 2012-10-17
         Statement:
           - Effect: Deny
-            Action: secretsmanager:GetSecretValue
+            Action:
+              - secretsmanager:GetSecretValue
+              - secretsmanager:DeleteSecret
```

Comment:
Should we add the Stack-specific Secrets as well? They have the Secret ID/Name `CfnStack/[stack name]/*`

Reply:
Isn't that already included?

```
              - !Sub arn:aws:secretsmanager:${AWS::Region}:${AWS::AccountId}:secret:CfnStack/*
```

Reply:
Yes, this is denying Get and Delete for `CfnStack/*`